site/content/en/docs/Getting started/first_kustomization.md

```diff
@@ -57,10 +107,259 @@ spec:
         app: nginx
     spec:
       containers:
-      - name: nginx
-        image: nginx:latest
+      - image: nginx
+        name: nginx
         ports:
         - containerPort: 80
+```
+
```

Comment:
It creates a bit of a harsh discontinuity to suddenly go from "here are some unchanged resources" to suddenly "bases and overlays"; I think it would be a good idea to include right here the other sections you were planning to add for the various fields (like namePrefix, labeels, etc), and have bases/overlays followed by components at the end. 

WDYT? We don't have to add all of that content in this PR, but maybe we can add a # TODO here.

Reply:
To be honest, I felt a bit the same about this when I was writing it and I was split between first applying the changes in one kustomization file and then moving on to overlays... I think you have a very valid point, so I could re-work it to show the changes without overlays (maybe just for the basic bits like nameprefix/suffix and namespace, and then have a section where we migrate it across to overlays and continue with examples for the other fields, what do you think?

Reply:
Personally I like the progressive approach, more or less what you have here. I would suggest removing the creation of the overlay directories until you need them, and not ever making them revert something they've done in a previous step. I'd also suggest making the headings imperative instead of questions, so that the progression is e.g.:
- Create your resource manifests
- Create your Kustomization
- Customize your resources
- Create variants using overlays
- Customize your overlays

Reply:
Thanks for the feedback, pushed some updates